### PR TITLE
View instructor code

### PR DESCRIPTION
**Pull request by melaniep518**

### CODE REVIEW

File: client/web_view/components/TreeNode/TreeNode.jsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import React, { Component, PropTypes } from 'react';
+import shortid from 'shortid';
```
Comment (samanthabretous) [inline client/web_view/components/TreeNode/TreeNode.jsx]: are we still using `shortid` in this file?

File: client/web_view/components/TreeNode/TreeNode.jsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    super(props);
+    this.state = {
+      visible: false,
+      currentPath: '/',
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
+    this.setState(prevState => ({
```
Comment (samanthabretous) [inline client/web_view/components/TreeNode/TreeNode.jsx]: Nice!! Way to use previous state.

File: client/web_view/components/Directory/Directory.jsx [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import { bindActionCreators } from 'redux';
-import shortid from 'shortid';
+import TreeNode from '../TreeNode/TreeNode';
```
Comment (samanthabretous) [inline client/web_view/components/Directory/Directory.jsx]: import from index.js

File: client/web_view/components/Directory/Directory.jsx [hunk 4]
```diff
@@ -24,12 +24,10 @@ class Directory extends Component {
 
   render() {
     const { directory } = this.state;
-    // shortid generates 7-14 digit unique ids, useful for looping functions
-    const files = Object.entries(directory).map(val => <li key={shortid.generate()}>{val[0]}</li>);
     return (
       <div>
         <h3>Instructor Directory</h3>
-        {files}
+        <TreeNode node={directory} />
```
Comment (samanthabretous) [inline client/web_view/components/Directory/Directory.jsx]: Do we need this file if it is only holding one component? Can the Treenode component be container

File: client/web_view/components/TreeNode/TreeNode.jsx (hunk 1 above)
Comment (melaniep518) [inline client/web_view/components/TreeNode/TreeNode.jsx]: yep

**Pull request merged**